app/test/std_sdk_test.go

```diff
@@ -220,6 +221,46 @@ func (s *StandardSDKIntegrationTestSuite) TestStandardSDK() {
 			},
 			expectedCode: abci.CodeTypeOK,
 		},
+		{
+			name: "create forbidden param proposal change",
+			msgFunc: func() (msgs []sdk.Msg, signer string) {
+				account := s.unusedAccount()
+				change := proposal.NewParamChange(stakingtypes.ModuleName, string(stakingtypes.KeyBondDenom), "stake")
+				content := proposal.NewParameterChangeProposal("title", "description", []proposal.ParamChange{change})
+				addr := getAddress(account, s.cctx.Keyring)
+				msg, err := oldgov.NewMsgSubmitProposal(
+					content,
+					sdk.NewCoins(
+						sdk.NewCoin(app.BondDenom, sdk.NewInt(1000000000))),
+					addr,
+				)
+				require.NoError(t, err)
+				return []sdk.Msg{msg}, account
+			},
+			// this parameter is protected by the paramfilter module, and we
+			// should expect an error. Due to how errors are bubbled up, we get
+			// this code despite wrapping the expected error,
+			// paramfilter.ErrForbiddenParameter
+			expectedCode: govtypes.ErrNoProposalHandlerExists.ABCICode(),
```

Comment:
[no change needed] but this is 😢

Reply:
true... we are checking for it via the unit tests however by checking for error messages directly, we might want to do that instead is these tests as well since the sdk picks a single code